lib/evss/aws_creds.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'net/http'
+
+module EVSS
+  class AwsCreds
+    METADATA_ENDPOINT = 'http://169.254.169.254/latest/meta-data/iam/security-credentials/CloudWatchRole/'
+
+    def self.fetch(fetch_from_remote = false)
```

Comment:
where would it fetch from if not remote? possibly consider changing this to be guard clause.

`return unless fetch_from_remote` instead of the if block.